### ISSUE TITLE
Does not work in frame

### ISSUE THREAD

**Issue by amitava82** [sanniassin/react-input-mask#182]
I'm trying to use this along with react-frame-component. My component is rendered in an iframe. Issue seems to be in following line
```
var isDOMNode = typeof window !== 'undefined' && input instanceof window.Element; // workaround for react-test-renderer
```
isDOMNode evaluates to false returning null and throwing `Uncaught TypeError: Cannot use 'in' operator to search for 'selectionStart' in null`

**Comment by dakiesse** [comment]
I have the same issue

**Comment by vladmelnikov** [comment]
seems i have same issue in last chrome version library throws error Cannot read property 'addEventListener' of null

**Comment by sanniassin** [comment]
Fixed and published as `3.0.0-alpha.1` on npm.
